src/domain/engine/Rule/rule.ts

```diff
@@ -0,0 +1,21 @@
+export class Rule {
+  private readonly _agesNumber: number = 3;
+  private readonly _availableWonders: number = 12;
+  private readonly _availablePerPlayer: number = 4;
```

Comment:
We need to be clear. Please rename it by **_availableWondersPerPlayer** and adapt the getter.